### PR TITLE
Add legalese to relinquish page

### LINKED ISSUE #3792: Make sure legalese is covered in benefit election/relinquishment
3) In the current VONAPP there is content regarding if the claimant relinquishes chapter 30, then they are only entitled to the amount of chapter 30 benefits they have remaining. We need to make sure that text is included.

- [ ] **“If electing chapter 33 in lieu of chapter 30, my months of entitlement under chapter 33 will be limited to the number of months of entitlement remaining under chapter 30 on the effective date of my election.”** 

It’s one of the bullets that was previously one of the 4 bullets on the benefits eligibility page (below). It could also be right after the chapter 30 on Benefits relinquishment (below). 

In addition, we need to put the following statement or similar after the 1607 selection on the relinquishment page. 

- [ ] **“(You are only eligible to relinquish 1607 if you received 1607 benefits for a semester, quarter, or term that included the date of November 24, 2015, or was the last semester, quarter, or term to end prior to November 24, 2015.)”**

### PR DESCRIPTION
**Pull request by alexose**
Fixes #3792.

image

### CODE REVIEW

**Comment by alexose** [comment]
This is ready to go after PR.  @akainic @jbalboni

**Comment by goldenmeanie** [comment]
LGTM

File: src/js/edu-benefits/utils/options-for-select.js [hunk 1]
```diff
@@ -470,9 +470,9 @@ const accountTypes = [
 
 const relinquishableBenefits = [
   { label: 'I’m only eligible for the Post-9/11 GI Bill', value: 'unknown' },
-  { label: 'Montgomery GI Bill (MGIB-AD, Chapter 30)', value: 'chapter30' },
+  { label: 'Montgomery GI Bill (MGIB-AD, Chapter 30)', additional: 'If you choose to give up MGIB-AD, you’ll get benefits only for the number of months you had left left under MGIB-AD.', value: 'chapter30' },
```
Comment (akainic) [inline src/js/edu-benefits/utils/options-for-select.js]: extra `left` here?

File: src/js/edu-benefits/components/benefits-eligibility/BenefitsRelinquishmentFields.jsx [hunk 2]
```diff
@@ -9,6 +9,46 @@ import { showRelinquishedEffectiveDate } from '../../utils/helpers';
...
+            <li>If you pick a future date, you can’t get benefits until then.</li>
+            <li>If your classes started less than 2 years ago, enter the date they began.</li>
+          </ul>
+        </div>
+      </div>
+    );
+
+    // nest DateInputs inside radio buttons
+    const options = relinquishableBenefits.map((d) => {
+      const option = Object.assign({}, d);
```
Comment (akainic) [inline src/js/edu-benefits/components/benefits-eligibility/BenefitsRelinquishmentFields.jsx]: This seems a little weird to me, to create a new object and then change the purpose of one of the properties (`additional`), but after walking through it I understand what you're doing. I think the design of this component and the way our components are structured makes this hard. Curious to see if @jbalboni has thoughts on a different way we could be handling this.

**Review by akainic — COMMENTED** [review]
Couple of thoughts.

File: src/js/edu-benefits/components/benefits-eligibility/BenefitsRelinquishmentFields.jsx [hunk 3]
```diff
@@ -19,33 +59,16 @@ export default class BenefitsRelinquishmentFields extends React.Component {
...
-              <div>
-                <ul>
-                  <li>Use today’s date unless you aren’t going to use your Post 9/11 GI Bill benefits until later.</li>
-                  <li>If you pick a future date, you can’t get benefits until then.</li>
-                  <li>If your classes started less than 2 years ago, enter the date they began.</li>
-                </ul>
-              </div>
+                  onValueChange={(update) => { this.props.onStateChange('benefitsRelinquished', update); }}/>
             </div>
+            <div></div>
```
Comment (alexose) [inline src/js/edu-benefits/components/benefits-eligibility/BenefitsRelinquishmentFields.jsx]: This div is needed for ExpandingGroup to work properly.  I know, it's a little weird...

File: src/js/edu-benefits/components/benefits-eligibility/BenefitsRelinquishmentFields.jsx [hunk 4]
```diff
@@ -9,6 +9,46 @@ import { showRelinquishedEffectiveDate } from '../../utils/helpers';
...
+            <li>Use today’s date unless you aren’t going to use your Post 9/11 GI Bill benefits until later.</li>
+            <li>If you pick a future date, you can’t get benefits until then.</li>
+            <li>If your classes started less than 2 years ago, enter the date they began.</li>
+          </ul>
+        </div>
+      </div>
+    );
+
+    // nest DateInputs inside radio buttons
+    const options = relinquishableBenefits.map((d) => {
```
Comment (alexose) [inline src/js/edu-benefits/components/benefits-eligibility/BenefitsRelinquishmentFields.jsx]: Sure, that's fine.  I noticed we don't have a ton of consistency in this arena.  Maybe we should add an eslint rule...

**Comment by alexose** [comment]
@goldenmeanie I've taken care of the ExpandingGroup problems:

image

**Comment by cweber** [comment]
Expanding groups look good now, but the content needs indented to align with the radio button label.

image

**Comment by goldenmeanie** [comment]
There is also a bounce when collapsing the group (select another radio). This requires a tweak to the margins [as is done here](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/sass/edu-benefits.scss#L120)

**Comment by cweber** [comment]
Looks good.

One minor animation bug still. If you select Chapter 30 and then 1606, there is a jump. It doesn't happen any other time you select 1606.

LGTM other than that though.

**Comment by alexose** [comment]
Playing around with margins to see if that's the issue...

**Comment by alexose** [comment]
I'm not sure there's much I can do to fix the jumpiness other than disable the close animation completely.

I just pushed a commit which does that... it looks good to my eyes, but you get final call.  Thanks!

**Comment by cweber** [comment]
I think that's fine for now. Get the language up is definitely more important than the close animation. I'll create a new issue (not a prioirty) for how we are handling close animations. Even in cases where it works it takes a lot of eyeballing to get it right for each case. It should probably be more flexible than that.

**Comment by alexose** [comment]
Great!  I'll revert the last commit and merge this after CI.

**Review by jbalboni — APPROVED** [review]
LGTM

**Pull request merged**